applications/deployments/route-based-deployment-strategies.adoc

```diff
@@ -21,7 +21,9 @@ for more on the basic strategy types.
 
 The most common route-based strategy is to use a _blue-green deployment_. The new version (the green version) is brought up for testing and evaluation, while the users still use the stable version (the blue version). When ready, the users are switched to the green version. If a problem arises, you can switch back to the blue version.
 
-A common alternative strategy is to use _A/B versions_ that are both active at the same time and some users use one version, and some users use the other version. This can be used for experimenting with user interface changes and other features to get user feedback. It can also be used to verify proper operation in a production context where problems impact a limited number of users.
+One alternative strategy is to use _A/B versions_ that are both active at the same time and some users use one version, and some users use the other version. This can be used for experimenting with user interface changes and other features to get user feedback. It can also be used to verify proper operation in a production context where problems impact a limited number of users.
```

Comment:
```suggestion
One alternative strategy is to use _A/B versions_ that are both active at the same time.  This allows some users to use _version A_, and others could use _version B_. This strategy could be used for experimenting with user interface changes or other features in order to get user feedback. It could also be used to verify proper operation in a production context where problems impact a limited number of users.
```

Reply:
@mburke5678, Thanks Michael, I applied those changes.